Katas/FizzBuzzTest/FizzTests.cs

```diff
@@ -15,6 +15,9 @@ public void Setup()
         [TestCase(1,"1")]
         [TestCase(2,"2")]
         [TestCase(4, "4")]
+        [TestCase(7, "7")]
+        [TestCase(11, "11")]
+        [TestCase(13, "13")]
```

Comment:
what led you to add these test cases?

Reply:
Nothing really i guess its not necessary right?